docs/development.md

```diff
@@ -57,6 +57,16 @@ If you're adding a new feature to one or more of the Java services (`ledgerwrite
 2. Re-run `skaffold dev` or `skaffold run` to trigger a Jib container build using Maven and the updated pom file. 
 
 
+## Generating your own JWT public key. 
```

Comment:
Can the app run without the JWT key currently?  I've only used the skaffold file which creates the JWT secret

Reply:
I think the deployments depend on the JWT secret to start up -but I actually think the app might be able to run without it? not sure.